### PR TITLE
Added yum install of libffi-devel to vagrant-setup.sh

### PR DESCRIPTION
**Pull request by csylvester-pivotal**
Change to vagrant/centos/vagrant-setup.sh to install the libffi-devel
package via yum. This is needed for compiling paramiko libraries.

### CODE REVIEW

File: vagrant/centos/vagrant-setup.sh [hunk 1]
```diff
@@ -18,6 +18,7 @@ sudo yum -y install epel-release
 sudo yum -y install htop
 sudo yum -y install perl-Env
 sudo yum -y install ccache
+sudo yum -y install libffi-devel
```
Comment (danielgustafsson) [inline vagrant/centos/vagrant-setup.sh]: dont we need the corresponding installation in the debian vagrantfile?

Comment (csylvester-pivotal) [inline vagrant/centos/vagrant-setup.sh]: Daniel,

Just checked and yes, the vagrant-setup.sh needs to be updated also:
line 9: `python-dev libxml2-dev pkg-config vim libbz2-dev  python-pip`
changed to:
          `python-dev libxml2-dev pkg-config vim libbz2-dev  python-pip libffi-dev`

However, there are other problems with the debian version that need more exploration.

/Craig

Comment (ascherbaum-pivotal) [inline vagrant/centos/vagrant-setup.sh]: What kind of problems did you identify? Is it worth opening an issue?

Comment (csylvester-pivotal) [inline vagrant/centos/vagrant-setup.sh]: Hi Andreas,

I submitted a PR for it. 

It caused the compilation failure of the paramiko package, which caused problems starting GP. 

Craig

Craig Sylvester
+1 703 362 0949

**Comment by danielgustafsson** [comment]
This change was covered in commit fb911c72322e675d08f3af9b7f04fdf341420b34, closing this one.

**Pull request closed without merging**